src/App.jsx

```diff
@@ -25,21 +25,8 @@ export const alex = {
 
 export const App = () => (
   <div className="App">
-    <section className="Person">
-      <h2 className="Person__name">My name is Misha</h2>
-      <p className="Person__age">I am 37</p>
-      <p className="Person__partner">Natasha is my wife</p>
-    </section>
-
-    <section className="Person">
-      <h2 className="Person__name">My name is Olya</h2>
-      <p className="Person__partner">Maksym is my husband</p>
-    </section>
-
-    <section className="Person">
-      <h2 className="Person__name">My name is Alex</h2>
-      <p className="Person__age">I am 25</p>
-      <p className="Person__partner">I am not married</p>
-    </section>
+    <Person person={misha} />
```

Comment:
You didn't import Person to app.jsx

![image](https://github.com/mate-academy/react_person/assets/84296041/750c057f-6165-4952-9da3-c71581ccef3b)
![image](https://github.com/mate-academy/react_person/assets/84296041/2c36c09b-b0a1-406d-9e73-2b3ca81ae785)